bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/Connection.java

```diff
@@ -402,6 +424,24 @@ public boolean tryRestoreLogin(@Nullable String data, @Nullable String overloade
         return loginTime;
     }
 
+    private @Nullable Authentication tryGetBootstrap() throws IOException, URISyntaxException {
+        String bootstrapResultJson = makeRequestAndReturnString(alexaServer + "/api/bootstrap");
+        boolean authenticated = bootstrapResultJson.contains("\"authenticated\":true");
```

Comment:
Hmm, I dont really like this string contains before parsing the json is there another way to do it better?

Reply:
If the user is not authenticated, the server just return a html page instead of a json, this would rise an exception in the json parser. But this is a common case, so I prefer to do this basic check before I run in a exception in the parser. But maybe I should make it not so restrictive, so I only check for the word authenticated and the enclosing {}, so that I do not run in trouble, if the json formatting is changed on the server side.

Reply:
But can't we use the content-type header or response code then, it seems like a more sensible approach to me

Reply:
Done